nnvm/python/nnvm/testing/yolo_detection.py

```diff
@@ -160,7 +160,7 @@ def draw_detections(im, dets, thresh, names, classes):
             if det['prob'][j] > thresh:
                 if category == -1:
                     category = j
-                labelstr.append(names[j])
+                labelstr.append(names[j] + "(" + str(round(det['prob'][j]*100, 2)) + ")")
```

Comment:
Yes, it will better for user to know the probability in the image itself. Either you add a % sign or keep probability as it is upto 3 decimals